opentech/apply/stream_forms/blocks.py

```diff
@@ -186,6 +186,38 @@ def get_field_kwargs(self, struct_value):
         return kwargs
 
 
+class GroupToggleBlock(FormFieldBlock):
+    required = BooleanBlock(label=_('Required'), default=True)
+    choices = ListBlock(
+        CharBlock(label=_('Choice')),
+        help_text='Please create only two choices for toggle. Additional choices will be ignored.'
```

Comment:
Please create only two choices for toggle. First choice will revel the group and the second hide it. Additional choices will be ignored.

Reply:
Currently, we can not use `max_num` option in ListBlock. There is one open PR in [Wagtail](https://github.com/wagtail/wagtail/pull/3914). Therefore handle this case by ignoring choices after 2. And if fewer choices are provided, default to Yes and No.

Reply:
Just realised comment is for help text. Updating PR.